docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -0,0 +1,75 @@
+[id='pro-running-the-published-repositories-{chapter}']
+= Introduction
+
+This section includes:
+
+. xref:pro-cloning-separate-repositories-{chapter}[Cloning the Git Repository]
+. xref:pro-installing-repository-npm-dependencies-{chapter}[Installing Core Dependencies]
+. xref:pro-starting-mongodb-and-redis-{chapter}[Starting MongoDB and Redis]
+. xref:pro-starting-the-core-server-{chapter}[Starting the Core Server]
+. xref:pro-running-the-mobile-and-portal-demo-applications-{chapter}[Running the Mobile and Portal Demo Applications]
+
+= Prerequisites
```

Comment:
Follow aproach from https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/63 
Let's not create sections in the toc for this but just standard headings.

Reply:
Done!

Reply:
via 04b294b